modern/src/common/formatter.js

```diff
@@ -80,3 +80,26 @@ export const formatVolume = (value, unit) => {
 export const formatHours = (value) => {
   return moment.duration(value).humanize();
 };
+
+const speedConverter = (value, unit) => {
```

Comment:
There's too much duplication with the functions above. Need to unify that. And also I think we should move converter functions into a separate file.

Reply:
done.